client/components/command-pallette/wpcom-command-pallette.tsx

```diff
@@ -34,12 +35,13 @@ export function CommandMenuGroup( {
 	return (
 		<Command.Group about="WPCOM">
 			{ commands.map( ( command ) => {
+				const itemValue = command.searchLabel ?? command.label;
 				return (
 					<Command.Item
 						key={ command.name }
-						value={ command.searchLabel ?? command.label }
+						value={ itemValue }
 						onSelect={ () => command.callback( { close, setSearch } ) }
-						id={ command.name }
+						id={ cleanForSlug( itemValue ) }
```

Comment:
I like this solution with `cleanForSlug`! TIL: https://developer.wordpress.org/block-editor/reference-guides/packages/packages-url/#cleanforslug